projects/plugins/jetpack/tests/e2e/specs/recommendations.test.js

```diff
@@ -6,9 +6,10 @@ import RecommendationsPage from '../lib/pages/wp-admin/recommendations';
 
 describe( 'Recommendations (Jetpack Assistant)', () => {
 	it( 'Recommendations (Jetpack Assistant)', async () => {
-		const recommendationsPage = await RecommendationsPage.visit( page );
+		let recommendationsPage;
 
 		await step( 'Navigate to the Recommendations module', async () => {
+			recommendationsPage = await RecommendationsPage.visit( page );
```

Comment:
Move `visit` into the step that is about navigating into the Recommendations module.